test/e2e-tests/support/options.go

```diff
@@ -0,0 +1,51 @@
+package support
+
+import (
+	"fmt"
+	"os"
+)
+
+const (
+	S3BucketNameEnvKey     = "S3_BUCKET"
+	RegistryRepoNameEnvKey = "REGISTRY_REPO"
+)
+
+var (
+	options *Options = nil
```

Comment:
I have some questions about this, after looking at how it's used:

- Options is quite a generic name. Is this expected to grow to hold all potential things that could be passed in or collected from the environment and used in the tests, or just pipeline run parameter values?
- This looks like an shared global variable between all tests: if we want to execute different PipelineRuns (either in the same test or in different tests), are we limited to using, for example, the same `target-imagerepo` for each?

Might not be worth thinking or doing anything about these questions until we need to, just thinking out loud.

Reply:
> Options is quite a generic name. Is this expected to grow to hold all potential things that could be passed in or collected from the environment and used in the tests, or just pipeline run parameter values?

My thinking was any options that can be used by the tests which are set by environment variables.

> This looks like an shared global variable between all tests: if we want to execute different PipelineRuns 

They are only shared if a tests wants to use them. Right now the structure I am thinking about and that I have talked to landon about is each test constructs one or more pipelineRuns and we see if it is a success.

If a tests does not want to use the options passed, it can just set that for a different value in the tests. The same way I am doing it https://github.com/opendatahub-io/ai-edge/pull/188/files#diff-a0b087c51d28b57a9b15240b642f35b5425c1bd662312f637ba56e176ecde3e1R92-R93

Reply:
Also Options is generic but that is how I was thinking of them, the same as arguments but instead passing them as args they are just set and env variables. If we want to change it that is fine with me. Maybe `TestingArguments`?

Reply:
I think this is fine for now. I wouldn't like to see it become a huge object that has lots of configurable knobs through environment variables. Let's just try to be mindful of what we might add to it, and not be afraid to use another approach if it suits us better in future :+1: